google/cloud/aiplatform/training_jobs.py

```diff
@@ -2662,7 +2835,9 @@ def _run(
         if model_display_name is None:
             model_display_name = self._display_name
 
-        model = gca_model.Model(display_name=model_display_name)
+        model = gca_model.Model(
+            display_name=model_display_name, encryption_spec=self.model_encryption_spec,
```

Comment:
This has a potential similar issue to where we reference this property that may pull the encryption spec key name from the global config at execution time of this potentially asynchronous method.

Reply:
Fixed